routs/events.js

```diff
@@ -0,0 +1,122 @@
+const fs = require('fs');
```

Comment:
I'd suggest restructuring the code a little bit. Right now this file handles `HTTP` layer as well as `data/csv` layer. I'd suggest separating those two areas of responsibility between different files/modules.